### PR TITLE
Added a number of bottles calculator, rounded up to the nearest integer.

### PR DESCRIPTION
**Pull request by i-jey**
First stab at contributing to a rust project. Would appreciate feedback on code-styling/standards if the reviewer has the time.
Was unsure of how the command line tool formatting should look like, so I went with a rust table style. Please let me know if there is some other way that it should be formatted, happy to make the change.

Thanks.

### CODE REVIEW

File: src/calculators/num_bottles.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            ("Gallon jug".to_string(), 3785.0),
+            ("5 liter mini keg".to_string(), 5000.0),
+        ]
+    }
+    /// Prints out the quantity of bottles needed to contain a volume given in mL
+    ///
+    /// # Arguments
+    /// * 'volume' - A f32 in milliliters of the volume to hold
+    ///
+    pub fn bottles_milliliters(&self, volume: f32) {
```
Comment (drodil) [inline src/calculators/num_bottles.rs:27]: Instead having different functions for different volume units, you could just use single one and convert it if necessary. There is also PR #25 which supports parsing the unit from string and converting it to any other unit easily using the measurements dependency (still waiting for merge though).

**Review by drodil — COMMENTED** [review]
Looks good start! Some duplications here and there and could really benefit of the #25 functionality. Also tests would be good addition as well as the CLI command (should be implemented in the src/commands folder, see examples there); this can be done in separate PR also.

**Comment by drodil** [comment]
#25 is now merged to master so please refactor code based on that, thanks!

**Comment by drodil** [comment]
Also about the coding standards and styling; you should use rustfmt to format the code. While it's not enforced (yet), it will be once #47 will be implemented.

**Comment by i-jey** [comment]
Added the refactored bottling calculator, working on the CLI command now, will push in a separate PR.

Cheers.

**Review by mlatief — DISMISSED** [review]
I assume something went wrong when rebasing this. To keep it simple, I suggest may be starting from current master and then add only your changes then push again to this branch or create a new PR from a new branch.

**Review by drodil — APPROVED** [review]

**Comment by drodil** [comment]
@all-contributors please add @i-jey for Code

**Pull request merged**